config/registry/aws/modules/aws-s3.yaml

```diff
@@ -38,10 +38,17 @@ inputs:
     description: A custom s3 policy json/yaml to add.
     default: null
   - name: cors_rule
-    user_facing: bool
+    user_facing: true
     validator: include('cors', required=False)
     description: A custom cors policy.
     default: null
+  - name: files
+    user_facing: true
+    validator: str(required=False)
+    description: |
+      The path (can be relative) to a directory holding files which you wish to upload to the s3 bucket.
```

Comment:
> The path (can be relative)

Relative to what? The cwd when the `opta` is run? The opta config file? opta itself? We should document what the path is relative to.

Reply:
Noticed this is documented in the module markdown doc, but we should document it here too.